src/main/java/seedu/bankwithus/Account.java

```diff
@@ -19,6 +20,7 @@ public Account(String name, String balance) {
         this.name = name;
         this.balance = balance;
         this.withdrawalChecker = new WithdrawalChecker();
+        this.saveGoal = new SaveGoal(0, "01-01-2001");
```

Comment:
Do note that there are 2 constructors for the Account class, so you might need to modify both of them rather than just one. I don't know how you plan to implement this feature though, so it is up to you to change it